tests/scripts/all.sh

```diff
@@ -557,6 +563,17 @@ cleanup
 cmake -D CMAKE_BUILD_TYPE:String=Check .
 if_build_succeeded tests/scripts/key-exchanges.pl
 
+if [ $TEST_CXX -ne 0 ]; then
+    msg "build: Unix make, gcc and g++ test" # ~ 30s
+    cleanup
+    make TEST_CPP=1
```

Comment:
Is there a reason for this being `CPP` rather than `CXX` like elsewhere? It's far from being a blocker, but it is possible to make it consistent, I'd take the opportunity.

Reply:
Agreed, CPP is the C preprocessor.

Reply:
Sure, I can change it, I just used it as an abbreviation of C++ (as in cppreference, not cxxreference).

Reply:
Update - Removed the flag alltogether.